### ISSUE TITLE
Missing feature : binding constraint marginal cost

### ISSUE THREAD

**Issue by flomnes** [AntaresSimulatorTeam/AntaREST#1549]
This feature has been available since Antares Simulator 8.4.0, but these new outputs are not visible in the "Results" tab in Antares Web.

Only outputs associated to links & areas are available.

![image](https://github.com/AntaresSimulatorTeam/AntaREST/assets/26088210/3d83843f-f293-4314-9c67-dbf660cb9b89)

**Comment by MartinBelthle** [comment]
Indeed, the files exist in the "Debug" tab but are not visible in the "Results" one. As the endpoint to retrieve these files already exists, I think there is nothing to do on the back office. Therefore, I added the label "front-end."
